### DISCUSSION TITLE
Replacing texts within the PDF

### DISCUSSION THREAD

**Discussion by edusachs** [qpdf/qpdf#945] [Q&A]
I would like to know if there is any qpdf command to replace words within a PDF?

For example, using sed, it would be something like: sed 's/text_old/text_new/g'

Thank you.

**Comment by jberkenbilt** [comment]
No, that is not something qpdf can or will likely ever be able to do. qpdf doesn't have any built-in functionality to work with the text in a PDF file. It only works with PDF at a structural level.
